sdk/cosmosdb/cosmos/test/internal/unit/getHeader.spec.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import assert from "assert";
+import { Constants } from "../../../src/common/constants";
+import { getHeaders } from "../../../src/request/request";
+import { CosmosHeaders, FeedOptions } from "../../../src";
+
+describe("Test ParallelizeCrossPartitionQuery header value", function () {
+  const mockedEndpoint = "https://localhost:8081";
+  function getHeadersFunc(feedOptions: FeedOptions): Promise<CosmosHeaders> {
+    return getHeaders({
+      clientOptions: { endpoint: mockedEndpoint },
+      defaultHeaders: null,
+      verb: null,
+      path: null,
+      resourceId: null,
+      resourceType: null,
+      options: feedOptions,
+      partitionKeyRangeId: null,
+      useMultipleWriteLocations: null,
+      partitionKey: null,
+    });
+  }
+  it("If maxDegreeOfParallelism > 1 then ParallelizeCrossPartitionQuery header's value should be true", async function () {
```

Comment:
can we put the header name here instead of ParallelizeCrossPartitionQuery header's value ?

Reply:
done